.pyspelling.yml

```diff
@@ -0,0 +1,25 @@
+spellchecker: aspell
+matrix:
+- name: beginner
+  sources:
+    - beginner_source/data_loading_tutorial.py
+  dictionary:
+    wordlists:
+      - tutorials-wordlist.txt
+  pipeline:
+  - pyspelling.filters.python:
+      group_comments: true
+  - pyspelling.filters.context:
+      context_visible_first: true
+      delimiters:
+        # Exclude figure rST tags
+        - open: '\.\.\s+(figure|literalinclude|)::'
+          close: '\n'
+        # Exclude Python coding directives
+        - open: '-\*- coding:'
+          close: '\n'
+  - pyspelling.filters.markdown:
+  - pyspelling.filters.html:
```

Comment:
Do we still need this filter?

Reply:
Yes, as it takes care of ignoring urls, codeblocks etc defined in markup